docs/next-release-notes.md

```diff
@@ -0,0 +1,517 @@
+
+https://github.com/apache/druid/pull/13334
+
+### When a Kafka stream becomes inactive, prevent Supervisor from creating new indexing tasks
+
+Added Idle feature to `SeekableStreamSupervisor` for inactive stream.
+
+https://github.com/apache/druid/pull/13144
+
+
+### Sampling from stream input now respects the configured timeout
+
+Fixed a problem where sampling from a stream input, such as Kafka or Kinesis, failed to respect the configured timeout when the stream had no records available. You can now set the maximum amount of time in which the entry iterator will return results.
+
+https://github.com/apache/druid/pull/13296
+
+### Streaming tasks resume on Overlord switch
+
+Fixed a problem where streaming ingestion tasks continued to run until their duration elapsed after the Overlord leader had issued a pause to the tasks. Now, when the Overlord switch occurs right after it has issued a pause to the task, the task remains in a paused state even after the Overlord re-election.
+
+https://github.com/apache/druid/pull/13223
+
+### Fixed Parquet list conversion
+
+Fixes an issue with Parquet list conversion, where lists of complex objects could unexpectedly be wrapped in an extra object, appearing as `[{"element":<actual_list_element>},{"element":<another_one>}...]` instead of the direct list. This changes the behavior of the parquet reader for lists of structured objects to be consistent with other parquet logical list conversions. The data is now fetched directly, more closely matching its expected structure.
+
+https://github.com/apache/druid/pull/13294
+
+### Introduced a tree type to flattenSpec
+
+Introduced a `tree` type to `flattenSpec`. In the event that a simple hierarchical lookup is required, the `tree` type allows for faster JSON parsing than `jq` and `path` parsing types.
+
+https://github.com/apache/druid/pull/12177
+
+## Operations
+
+### Compaction
+
+Compaction behavior has changed to improve the amount of time it takes and disk space it takes:
+
+- When segments need to be fetched, download them one at a time and delete them when Druid is done with them. This still takes time but minimizes the required disk space.
+- Don't fetch segments on the main compact task when they aren't needed. If the user provides a full `granularitySpec`, `dimensionsSpec`, and `metricsSpec`, Druid skips fetching segments.
+
+For more information, see the documentation on [Compaction](https://druid.apache.org/docs/latest/data-management/compaction.html) and [Automatic compaction](https://druid.apache.org/docs/latest/data-management/automatic-compaction.html).
+
+https://github.com/apache/druid/pull/13280
+
+### New metric for segments
+
+`segment/handoff/time` captures the total time taken for handoff for a given set of published segments.
+
+https://github.com/apache/druid/pull/13238 
+
+### Idle configs for the Supervisor
```

Comment:
I think we should explain to the reader that it's now possible to set supervisor to ideal. This is useful in cases of freeing up slots such that auto-scaling can kick in more effectively.